### DISCUSSION TITLE
App runs ok with dev but 404's with start

### DISCUSSION THREAD

**Discussion by o-t-w** [vercel/next.js#28878] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

16.8.0

### What browser are you using?

Latest stable Chrome

### What operating system are you using?

Windows 10

### How are you deploying your application?

Localhost

### Describe the Bug

When I run npm run build followed by npm run start I get a 404 for my pages/index.js page (which is the only page in my app).
npm run dev works fine.

### Expected Behavior

I should be able to see my app on localhost:3000

### To Reproduce

Not sure

**Comment by balazsorban44** [comment]
> To Reproduce "Not sure".

This is practically impossible to debug for anyone.

A proper reproduction is needed.

@o-t-w could you create a reproduction or at least paste in your `pages/index.js` page and your `next.config.js`?

**Comment by o-t-w** [comment]
To reproduce: clone the following repo (maybe on Windows 10 as that might make a difference?) and then run npm run build and npm run start and then look in a web browser. 
https://github.com/o-t-w/immo-search/

**Comment by o-t-w** [comment]
No idea if it’s related but I had also run into this issue earlier today https://stackoverflow.com/questions/58365151/hooks-error-invalid-hook-call-using-nextjs-or-reactjs-on-windows/58365152#58365152

**Comment by balazsorban44** [comment]
Unfortunately, the issue is not reproducible given your linked repository. (Just tried it locally, and everything worked correctly). Try removing the `.next` folder and re-run `npm run build` and `npm run start`

Do you have some screenshots showing the different pages in dev/prod build?

The more info you provide from the get-go, the easier it is to help you.

**Reply by o-t-w** [reply]
This is the message I get when I do `npm run start` 
```{"message":"Route GET:/ not found","error":"Not Found","statusCode":404}```
![image](https://user-images.githubusercontent.com/29547922/132395808-ccf06b0d-79e9-49d2-8570-c76c7a0fc3a7.png)

**Reply by o-t-w** [reply]
I tried deleting the `.next` and rebuilding like you suggested and it didn't work. Then I deleted it and rebuilt again and it _did_ work. Honestly don't know why Tim removed this from the issues though - clearly seems like a bug  ¯\_(ツ)_/¯

Thanks for the help @balazsorban44

**Reply by balazsorban44** [reply]
It is not removed, just MOVED. Without a clear reproduction or description, it is at best a user error, unfortunately, but at least not a confirmed bug 🐛. Especially if it actually worked after a clean rebuild. 👀

Is the behavior consistent across browsers? Try incognito modes in different ones to be sure it's not a browser extension issue.

**Reply by wpickachu** [reply]
I also do have this issue in my personal website.
